tools/xcodeproj_shims/installers/indexstores.sh

```diff
@@ -6,12 +6,18 @@ set -euo pipefail
 
 echo "Start remapping index files at `date`"
 
-FOUND_INDEXSTORES=`grep -o 'command_line: "\(.*\.indexstore\)' $BAZEL_BUILD_EVENT_TEXT_FILENAME | sed 's/.*command_line: "//' | uniq`
+FOUND_INDEXSTORES=`grep -o 'command_line: "\(.*\.indexstore\)' $BAZEL_BUILD_EVENT_TEXT_FILENAME | sed 's/.*command_line: "//' | uniq` || true
```

Comment:
QQ: If the command to `FIND_INDEX_STORES` fails what will it loop over?

Reply:
Well the effect is that it will terminate existing `index-import` processes - this is a big improvement over locking up the machine 👍 !